tools/release/publish-release.ts

```diff
@@ -130,6 +130,9 @@ class PublishReleaseTask extends BaseReleaseTask {
     console.info(chalk.green(chalk.bold(`  ✓   Published all packages successfully`)));
     console.info(chalk.yellow(`  ⚠   Please draft a new release of the version on Github.`));
     console.info(chalk.yellow(`      ${newReleaseUrl}`));
+
+    // Remove file at ~/.npmrc after release is complete.
+    spawnSync('rm', ['~/.npmrc'], {cwd: this.projectDir, stdio: 'inherit', shell: true});
```

Comment:
Why not use NodeJS unlinkSync? Not a big deal though, so I'm happy leaving it like that.

Reply:
Done, didn't even know it existed.